Cheetah/c/_namemapper.c

```diff
@@ -255,7 +300,8 @@ static PyObject *namemapper_valueForName(PYARGS)
 
     createNameCopyAndChunks();
 
-    theValue = PyNamemapper_valueForName(obj, nameChunks, numChunks, executeCallables, useDottedNotation);
+    /* passing 0 for isFirst, VFN is only called like VFN(VFFSL(...), ...) */
```

Comment:
More importantly, if anyone uses this stuff, they will get strictly too many errors.